frontend/dockerfile/dockerfile_test.go

```diff
@@ -5011,15 +5095,24 @@ COPY --from=s1 unique2 /
 }
 
 func testPlatformArgsImplicit(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	f := getFrontend(t, sb)
 
-	dockerfile := []byte(fmt.Sprintf(`
+	dockerfileTemplate := integration.UnixOrWindows(
```

Comment:
nit: for semantics, I think  you can still just call it `dockerfile`, as templates are [quite different](https://www.digitalocean.com/community/tutorials/how-to-use-templates-in-go). And then just re-use the variable next at L5115.